src/tests/playwright/pages/tests.json

```diff
@@ -0,0 +1,51 @@
+{
+  "benchmark": [
```

Comment:
What's the difference between "benchmark" and "performance"?

Reply:
This group of tests can be focused on correctness too compared to just pure performance

Reply:
Then I guess the right term is "conformance" testing. We can fix that later.